client/app/app.js

```diff
@@ -109,7 +109,7 @@ angular
 
         $translateProvider.useMissingTranslationHandler("translateMissingTranslationHandler");
         $translateProvider.useLoaderCache(true);
-        $translateProvider.useSanitizeValueStrategy("sanitizeParameters");
+        $translateProvider.useSanitizeValueStrategy("escapeParameters");
```

Comment:
Fixes an issue with UTF8 chars encoding while keeping security on: https://angular-translate.github.io/docs/#/guide/19_security

Reply:
Are you sure that won't break nothing?

Reply:
I don't know:
* it's the value recommended by the documentation
* `"sanitizeParameters"` already breaks some characters